.cypress/integration/ad/dashboard/ad_dashboard.spec.ts

```diff
@@ -19,21 +19,21 @@ import { buildAdAppUrl } from '../../../utils/helpers';
 context('AD Dashboard', () => {
   it('Empty dashboard - no detector index', () => {
     cy.mockGetDetectorOnAction('no_detector_index_response.json', () => {
-      cy.visit(buildAdAppUrl(DASHBOARD));
+      cy.visitWithAuth(buildAdAppUrl(DASHBOARD));
```

Comment:
What if people want to test without security plugin enabled?

Reply:
> What if people want to test without security plugin enabled?

So it will still work on a security-disabled server. Can see in the latest run that it works on both security enabled/disabled https://github.com/ohltyler/anomaly-detection-kibana-plugin/actions/runs/187512306